.github/workflows/regression-test-TT02.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       uses: grafana/k6-action@v0.3.1
       with:
         filename: test/k6/src/tests/instances.js
-        flags: -e runFullTestSet=true -e env=${{ vars.ENV }} -e org=${{ vars.ORG }} -e app=${{ vars.APP }} -e userName=${{ secrets.USE_CASE_USER_NAME}} -e userPassword=${{ secrets.USER_PASSWORD}} -e apimSubsKey=${{ secrets.APIM_SUBSKEY }} -e mpClientId=${{ secrets.MP_CLIENT_ID }} -e mpKid=${{ secrets.MP_KID }} -e encodedJwk=${{ secrets.MP_ENCODEDJWK }}
+        flags: -e runFullTestSet=true -e env=${{ vars.ENV }} -e pid=${{ secrets.PERSON_NUMBER }} -e partyId=${{ secrets.PARTY_ID }} -e orgNumber=${{ secrets.ORG_NUMBER }} -e org=${{ vars.ORG }} -e app=${{ vars.APP }} -e userName=${{ secrets.USE_CASE_USER_NAME}} -e userPassword=${{ secrets.USER_PASSWORD}} -e apimSubsKey=${{ secrets.APIM_SUBSKEY }} -e mpClientId=${{ secrets.MP_CLIENT_ID }} -e mpKid=${{ secrets.MP_KID }} -e encodedJwk=${{ secrets.MP_ENCODEDJWK }}
```

Comment:
Same goes here. Why is partyId required?

Reply:
https://github.com/Altinn/altinn-storage/pull/381#discussion_r1611101856